charts/cloudzero-agent/values.yaml

```diff
@@ -48,11 +48,11 @@ prometheusConfig:
     additionalScrapeJobs: []
 
 kube-state-metrics:
-  enabled: false
+  enabled: true
```

Comment:
@josephbarnett You mentioned in an offline conversation that it should be okay to set these to `true`, by default so that we don't have to pass them in the `test-chart`. Let me know if you still feel the same.

Reply:
yeah, it makes sense to me personally to have it set to true, especially since it's not a daemonset like node exporter. I think the recommendation to have them off by default came from Will. which I get, too; the error case if you deploy KSM when you already have one running is probably (maybe?) worse than if you don't have it at all.

so I guess I could go either way. but overall, I'd vote for not setting it to true, because it would be a breaking change. BUT, removing node-exporter is a pretty big change too - maybe now's the time we got for a major release?

Reply:
Awesome, thanks for clarifying.

>  maybe now's the time we got for a major release?

That makes sense to me.